src/pages/talks/[slug].tsx

```diff
@@ -136,7 +136,7 @@ const Talk: FunctionComponent<LessonProps> = ({initialLesson}) => {
         <script src="//cdn.bitmovin.com/player/web/8/bitmovinplayer.js" />
       </Head>
       <div>
-        <div className="bg-black -mt-3 sm:-mt-5 sm:-mx-8 -mx-5">
+        <div className="bg-black -mt-3 sm:-mt-5">
```

Comment:
This left and right margins are not included in the player size, so it becomes wider than 100% and therefore there is a horizontal scroll